### ISSUE TITLE
[Feature Request] More firewall examples

### ISSUE THREAD

**Issue by lfjdhr123** [polycube-network/polycube#329]
## Is your feature request related to a problem? Please describe.
When I went through [the firewall instruction](https://polycube-network.readthedocs.io/en/latest/services/pcn-firewall/firewall.html) and check out [the examples](https://github.com/polycube-network/polycube/tree/master/src/services/pcn-firewall/examples/), not really helped me to know how to use the firewall.
After I create my firewall by `polycubectl firewall add myFW`, the status show INGRESS and EGRESS are dropped but I still can access the machine normally.

## Describe the solution you'd like
After I did the following steps:
```
1. run "polycubectl firewall add myFW" to make a new firewall rule
2. run "polycubectl firewall show myFW" to make sure both INGRESS and EGRESS are dropped
2. run "polycubectl attach myFW enp0s31f6" to attach the firewall rule to my network interface
```
The firewall works well.

**Comment by acloudiator** [comment]
Thanks @lfjdhr123 for reporting this. We'll review and update our documentation accordingly.

**Comment by frisso** [comment]
Closed by #332
